goccm.go

```diff
@@ -35,7 +40,7 @@ type (
 		allDoneCh chan bool
 
 		// The close flag allows we know when we can close the manager
-		closed bool
+		closed atomic.Bool
```

Comment:
atomic.Bool is only available since Go 1.19. Also, other contributor presented another solution using channel. Could you take a look: https://github.com/zenthangplus/goccm/pull/9

Reply:
just fixed it another commit.